tools/hexfiend/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+SRCDIR="$(dirname -- ${BASH_SOURCE[0]:-$0})"
+DSTDIR="$HOME/Library/Application Support/com.ridiculousfish.HexFiend"
+
+if [[ -d "/Applications/Hex Fiend.app" ]]; then
+  echo "=== HexFiend already installed"
+else
+  echo "=== Installing HexFiend ..."
+  brew install --cask hex-fiend
+fi
+
+
+echo
+echo "=== Copying templates ..."
+cp -a "$SRCDIR/Templates" "$DSTDIR/"
```

Comment:
On my system, installing `hex-fiend` did not create the `com.ridiculousfish.HexFiend` directory under `~Library/Application Support`. So this copy command wound up copying the `Template` directory to be that directory. I.e.
```
$ ls ~/Library/Application\ Support/com.ridiculousfish.HexFiend/
BSON		WT		hexparse
```

Running `hexparse` from here doesn't work. I assume because the template aren't in the templated directory:

```
$ ./hexparse WT ~/notes/WT-10693/RUNDIR.69/WiredTiger.wt
Can't find template: WT. Available templates:
$
```

Reply:
@keitharnoldsmith thanks for trying it out! I have fixed these. Can you please try again?